SpatialGDK/Source/SpatialGDK/Private/Interop/RPCExecutor.cpp

```diff
@@ -0,0 +1,117 @@
+﻿// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#include "Interop/RPCExecutor.h"
+
+#include "Interop/Connection/SpatialTraceEventBuilder.h"
+#include "Interop/SpatialPlayerSpawner.h"
+#include "Interop/SpatialReceiver.h"
+#include "Interop/SpatialSender.h"
+#include "Utils/RepLayoutUtils.h"
+
+using namespace SpatialGDK;
+
+RPCExecutor::RPCExecutor(class USpatialNetDriver* InNetDriver)
```

Comment:
Don't forward declare classes inside a function arguments. Better to do it at the top of the file if necessary, however this type needs to be complete in this file so it's not necessary here.